portfolio/public/index.html

```diff
@@ -1,43 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Change the contents of this file for your site defaults, dont know if you knew that

Reply:
Also, you should make the inner `portfolio` folder the source of the repo, but nbd